### DISCUSSION TITLE
Show directories first

### DISCUSSION THREAD

**Discussion by jupblb** [srid/emanote#164] [Ideas]
Is it possible to make emanote show directories first (in the navigation pane) when using the default layout?

For example: let's take a look at https://emanote.srid.ca/tips. According to the feature request, it should be possible to make "Javascript behaviours" appear before "Adding images".

**Comment by srid** [comment]
It is not documented yet but note that you can specify a custom order for sidebar using the `order` YAML property (it is a number). 

Example, https://raw.githubusercontent.com/srid/emanote/master/docs/guide.md

For any `${folder}` you can create a `${folder}.md` and then specify the YAML property in it.

**Reply by srid** [reply]
But yes, having an option to show directories first may nevertheless be useful to have.

**Reply by jupblb** [reply]
This is absolutely fantastic and definitely meets my needs. Thank you!
